### ISSUE TITLE
Create a networking package

### ISSUE THREAD

**Issue by WanWizard** [fuelphp/fuelphp#32]
This package can contain the current 1.x FTP class, and also provide interfaces to PHPSecLib's SFTP and SSH functionality.

**Comment by dercoder** [comment]
What do you think about using https://github.com/thephpleague/shunt instead of "PHPSecLib"?

**Comment by WanWizard** [comment]
Possible. The only reason for suggesting PHPSecLib was that it's already used elsewhere, so you wouldn't need another package.

With the availability of composer and all it's packages, the question becomes whether or not the framework should include this functionality, or if we should leave the choice to the developer (and possibly only suggest a package)? I think the latter...

**Comment by dercoder** [comment]
Ok, i understand. 

But i think it would be better if the Network package can the basic network stuff like
ftp, sftp, scp and ssh.

If you are also thinking about creating socket servers or clients i recommend.
http://reactphp.org/

**Comment by sagikazarmark** [comment]
I agree with @WanWizard: it is not something that belongs to a framework nowadays.

**Comment by dercoder** [comment]
Ok, its clear now ;)
